AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -320,6 +320,7 @@ usrowing.org,stevensducks.com##.sidearm-adblock
 liga.net#@##bottomAd
 abcnyheter.no##.fc-ab-root
 doodlr.io#%#//scriptlet("set-constant", "isAdBlockActive", "false")
+duolingo.com##div[data-test="player-end-carousel"] > div[class]:nth-child(2)
```

Comment:
or we can add 
```
duolingo.com#$#.pub_300x250.pub_300x250m.pub_728x90.text-ad.textAd.text_ad.text_ads.text-ads.text-ad-links { display:block!important; }
@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,domain=duolingo.com
duolingo.com#@#.adsbygoogle
```

Reply:
I guess we can use this in Annoyances `duolingo.com#?##root div[data-test="player-end-carousel"] > div[class]:has(> div[class] a[href*="-disable-my-ad-blocker-"][target="_blank"])`
These rules should be enough for content blockers
```
||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=duolingo.com
duolingo.com#@#.adsbygoogle
```

Reply:
agree